_includes/faq.md

```diff
@@ -0,0 +1,23 @@
+## FAQ
+
+### Why is there no agenda?
+
+We call this "Everybody talks everybody listens"™️.
+Each participant is given a slot to present the subject of their choosing. 
+Depending on the attendance and session the slot may be between 5 and 30 minutes long.
```

Comment:
I would update to "session topic" to clarify that the topic impacts how long the slot will be.

Reply:
@switzersc mhm. By "session" I meant something else. Group of presentations. A block perhaps?

Reply:
Ohh I see. I think you're right - the best word probably is session.

Reply:
Hrm... we've used "session talks" for the 30 minute length ones in the past. How about...
"Depending on the attendance we'll have speaking slots between 5 and 30 minutes long."